src/AdminForm.php

```diff
@@ -1369,6 +1371,7 @@ private function buildOptionsTab() {
       }
       $this->form['additional_options']['confirm_subscription']['#description'] .= implode(', ', $ml) . '</em>';
     }
+
     else {
```

Comment:
Be careful adding lines/space -> best to delete this extra line here.

Reply:
@KarinG Sounds good, I'll remove that extra line!